prow/config.yaml

```diff
@@ -8000,6 +8000,40 @@ periodics:
 
 - interval: 30m
   agent: kubernetes
+  name: ci-kubernetes-e2e-gce-alpha-api
+  spec:
+    containers:
+    - args:
+      - --timeout=70
```

Comment:
this timeout need to be 20m longer than the one from config.json - either extend this one or shorten the one there